lib/charms/grafana_agent/v0/cos_agent.py

```diff
@@ -677,15 +721,7 @@ def metrics_jobs(self) -> List[Dict]:
         """Parse the relation data contents and extract the metrics jobs."""
         scrape_jobs = []
         if data := self._principal_unit_data:
-            jobs = data.metrics_scrape_jobs
-            if jobs:
-                for job in jobs:
-                    job_config = {
-                        "job_name": job["job_name"],
-                        "metrics_path": job["path"],
-                        "static_configs": [{"targets": [f"localhost:{job['port']}"]}],
-                    }
-                    scrape_jobs.append(job_config)
+            scrape_jobs = data.metrics_scrape_jobs
 
```

Comment:
This is not backwards-compatible, does it take bumping the LIBAPI version or just handle the old format as well?